tests/integ/scala/test_udf_suite.py

```diff
@@ -795,10 +795,22 @@ def variant_int_output_udf(_):
     def variant_float_output_udf(_):
         return 1.1
 
-    Utils.check_answer(
-        TestData.variant1(session).select(variant_float_output_udf("num1")).collect(),
-        [Row("1.1")],
-    )
+    if is_in_stored_procedure():
+        # current behavior in reg env: https://snowflakecomputing.atlassian.net/browse/SNOW-726724
+        Utils.check_answer(
+            TestData.variant1(session)
+            .select(variant_float_output_udf("num1"))
+            .collect(),
+            [Row("1.100000000000000e+00")],
```

Comment:
will this be a BCR to stored proc customer?

Reply:
no, this only affect dev/reg/qa today. That's why we are affected.